ddl/ddl_api.go

```diff
@@ -3422,6 +3423,20 @@ func (d *ddl) DropIndex(ctx sessionctx.Context, ti ast.Ident, indexName model.CI
 	return errors.Trace(err)
 }
 
+func countOfIndexContainColumn(tblInfo *model.TableInfo, colName string) int {
+	count := 0
+	for _, idx := range tblInfo.Indices {
+		for _, c := range idx.Columns {
+			if c.Name.L != colName {
+				continue
+			}
+			count++
+			break
```

Comment:
I think we can change this code as follows:
```
			if c.Name.L == colName {
				count++
				break
			}
```

Reply:
Great, Done.